### DISCUSSION TITLE
Eigenfunctions

### DISCUSSION THREAD

**Discussion by MSAfify** [danielver02/ALPS#164] [Q&A] [answered]
Hi Prof. Daniel,
Please let me know the format of the output eigenfunctions that are included in the solution.

Thanks!
Mahmoud

**Comment by danielver02** [comment, marked as answer]
Hi Mahmoud,

Please have a look at the documentation under 
[https://danielver02.github.io/ALPS/page/output.html](https://danielver02.github.io/ALPS/page/output.html)

The output format is explained there.

Best wishes
-Daniel

**Reply by MSAfify** [reply]
Hi Prof. Daniel,
Great thanks!
Mahmoud
